pkgs/os-specific/linux/lxc/default.nix

```diff
@@ -57,6 +57,7 @@ stdenv.mkDerivation rec {
   installFlags = [
     "localstatedir=\${TMPDIR}"
     "sysconfdir=\${out}/etc"
+    "bashcompdir=\${out}/etc/bash_completion.d"
```

Comment:
Bash completion files should no longer go to `etc` since these are no configuration files. Instead, `$out/share/bash-completion/completions/lxc` would be the proper place.